taqueria-plugin-tezos-client/common.ts

```diff
@@ -0,0 +1,39 @@
+import { RequestArgs } from '@taqueria/node-sdk/types';
+import { join } from 'path';
+
+export interface TypeCheckOpts extends RequestArgs.ProxyRequestArgs {
+	sourceFile: string;
+}
+
+export interface SimulateOpts extends RequestArgs.ProxyRequestArgs {
+	sourceFile: string;
+	storage?: string;
+	param: string;
+	entrypoint?: string;
+}
+
+// To be used for the main entrypoint of the plugin
+export type IntersectionOpts = TypeCheckOpts & SimulateOpts;
+
+// To be used for common functions in this file
+type UnionOpts = TypeCheckOpts | SimulateOpts;
+
+export const FLEXTESA_IMAGE = 'oxheadalpha/flextesa:latest';
+export const GLOBAL_OPTIONS =
```

Comment:
Again, despite this being temporary, if it's temporary long enough, and a node goes down, then users might run into the same situation and dependent on us releasing a fix. To mitigate against that, perhaps we'd be best to have these things overridable via environment variables. So perhaps we could use:

```
const endpoint = process.env['TAQ_TEZOS_CLIENT_RPC'] ?? 'https://ghostnet.ecadinfra.com'`

const protocol = process.env['TAQ_TEZOS_CLIENT_PROTOCOL'] ??
'PtKathmankSpLLDALzWw7CGD2j2MtyveTwboEYokqUCP4a1LxMg'
```

Reply:
Good idea

Reply:
Awesome, great suggestion @mweichert!